lib/state/wallet.ts

```diff
@@ -32,28 +42,29 @@ export type UseWallet = WalletState &
      * @returns void
      */
     selectAccount: (account: string) => void;
-    /**
-     * Get the active signer for transactions.
-     * @returns KeyringPairOrExtSigner | null
-     */
-    getActiveSigner: () => KeyringPairOrExtSigner | null;
     /**
      * Disconnect the wallet.
      * @returns void
      */
     disconnectWallet: () => void;
     /**
-     * Whether the wallet is nova wallet.
+     * Set if proxy execution is enabled.
      */
-    isNovaWallet: boolean;
+    setProxyFor: (real: string, conf: ProxyConfig) => void;
   };
 
+export type ProxyConfig = {
+  enabled: boolean;
+  address: string;
+};
+
 /**
  * State type of user wallet config.
  */
 export type WalletUserConfig = Partial<{
   walletId: string;
   selectedAddress: string;
```

Comment:
So selectedAddress is the address selected by the user in the account button and will allways be the same as `wallet.activeAccount.address` both when using proxy and when not. `wallet.realAddress` will be the same as `wallet.activeAccount.address` when not using proxy and when using proxy it will be the account that `wallet.activeAccount.address` is proxying for 😅 

The user config is just what the user as explicitly selected. selectedAddress can even be empty when the user first connects the wallet, in that case `wallet.activeAccount`will be the first address in the extensions list of accounts.

So for reference. `realAddress` should allways be used when reading data concering the users markets, trades etc. and wallet.activeAccount.address should be used to determine transaction fees since that is the signing account. `selectedAddress` is no longer exposed on wallet but is only used internally to explicitly manager user settings.

Reply:
tldr. yes it is used/needed but only internally in the wallet.